specification/network/resource-manager/Microsoft.Network/preview/2022-06-01-preview/networkVerifier.json

```diff
@@ -0,0 +1,1453 @@
+            "in": "path",
+            "name": "analysisIntentName",
+            "description": "The name of the Analysis Intent resource",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "analysisRunName",
+            "description": "The name of the Analysis Run resource",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "The Analysis Run resource object to create/update",
+            "schema": {
+              "$ref": "#/definitions/AnalysisRun"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "AnalysisRun created and will complete asynchronously",
+            "schema": {
+              "$ref": "#/definitions/AnalysisRun"
+            }
+          },
+          "404": {
+            "description": "Parent resource not found",
+            "schema": {
+              "$ref": "./network.json#/definitions/Error"
+            },
+            "x-ms-error-response": true
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "#/definitions/ResourceProviderDefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "AnalysisRun_Create": {
+            "$ref": "./examples/AnalysisRun_Create.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "AnalysisRun"
+        ],
+        "summary": "Deletes the specific Analysis Run resource asynchronously",
+        "operationId": "AnalysisRun_Delete",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "in": "path",
+            "name": "networkVerifierName",
+            "description": "The name of the Network Verifier resource",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "analysisIntentName",
+            "description": "The name of the Analysis Intent resource",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "analysisRunName",
+            "description": "The name of the Analysis Run resource",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
```

Comment:
>responses [](http://example.com/codeflow?start=9&length=9)

Add 200. Please correct all response status codes in the PR:
GET => 200, default
PUT => 200, 201, default
DELETE => 200, 202 (if long-running), 204, default
PATCH => 200, 202 (if long-running), default